blockchain/genesis/genesis.go

```diff
@@ -314,9 +316,9 @@ func New() (Genesis, error) {
 }
 
 // SetGenesisTimestamp sets the genesis timestamp
-func SetGenesisTimestamp(ts int64) {
+func (g *Genesis) SetGenesisTimestamp() {
```

Comment:
if we try to set a global parameter:
```
func SetGenesisHash(g *Genesis) {
  ...
  genesisHash = g.Hash()
  ...
}

Reply:
ok, i think this way does understand better

Reply:
fixed in latest push